From e5cb27f5209ac2db29b4ef8531465003d35eda1e Mon Sep 17 00:00:00 2001
From: Piotr Maziarz <piotrx.maziarz@intel.com>
Date: Wed, 13 Mar 2019 11:32:27 +0100
Subject: [PATCH 1356/1494] ASoC: Intel: Skylake: make function static inline

Fix empty function definition used when debugfs is not builded.

Currently module does not compile without debugfs.
Functions must be static to build and inline to avoid warnings.

Change-Id: I7adac1f84b18fcee52f3947274571f7346c2911c
Signed-off-by: Piotr Maziarz <piotrx.maziarz@intel.com>
Signed-off-by: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
---
 sound/soc/intel/skylake/skl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/skl.h b/sound/soc/intel/skylake/skl.h
index c900bdf..1f459be 100644
--- a/sound/soc/intel/skylake/skl.h
+++ b/sound/soc/intel/skylake/skl.h
@@ -266,7 +266,7 @@ static inline struct nhlt_specific_cfg
 {
 	return NULL;
 }
-void skl_dbg_event(struct skl_sst *ctx, int type)
+static inline void skl_dbg_event(struct skl_sst *ctx, int type)
 {
 }
 #endif
-- 
2.7.4

